pandas/tests/io/test_sql.py

```diff
@@ -1787,10 +1787,12 @@ def test_read_procedure(self):
 
         connection = self.conn.connect()
         trans = connection.begin()
+        
+        import pymysql
```

Comment:
You will have to move this at the top of the file.

Also, we started to validate the order of the imports. Many files are still excluded, and won't make the CI file yet, but it's worth having it in the right position. You can check:

https://github.com/pandas-dev/pandas/blob/master/doc/source/contributing.rst#import-formatting

If this file is excluded, you can run `isort` and remove it from `setup.cfg`.